### PR TITLE
Add deprecation infromation for DCOS History

### PR DESCRIPTION
**Pull request by janisz**
## Description
https://jira.mesosphere.com/browse/DCOS-51011

Refs: https://github.com/mesosphere/dcos-docs-site/commit/7be158d9b7169e8f3defb39c5336bf2f37f69122#diff-777594e08c04f42635a03a0ef6d89b83R210

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by janisz** [comment]
:wave: @drewvanstone @br-lewis @makkes  Can you review, please?

File: pages/1.13/monitoring/logging/aggregating/splunk/index.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ For each master node in your DC/OS cluster:
         -u dcos-exhibitor.service          \
         -u dcos-gen-resolvconf.service     \
         -u dcos-gen-resolvconf.timer       \
-        -u dcos-history.service            \
+        -u dcos-history.service            \ # deprecated
```
Comment (drewvanstone) [inline pages/1.13/monitoring/logging/aggregating/splunk/index.md]: I don't think we should add the deprecated comment here, personally. It looks a bit weird on the site. I think if the dcos-history service itself is documented as deprecated, then we can just live this as-is here, and remove it later. This page is about Splunk, not really about the history service.

File: pages/1.13/monitoring/logging/quickstart/index.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ You can view logs from tasks or the host subsystem with the `dcos node log` comm
         dcos-exhibitor.service
         dcos-gen-resolvconf.service
         dcos-gen-resolvconf.timer
-        dcos-history.service
+        dcos-history.service          # deprecated
```
Comment (drewvanstone) [inline pages/1.13/monitoring/logging/quickstart/index.md]: I don't think we should add this comment here either. Again, kinda looks weird.

File: pages/1.13/overview/concepts/index.md [hunk 3]
```diff
@@ -242,7 +242,7 @@ A DC/OS component is a DC/OS system service that is distributed with DC/OS.
 
 Most components run on the master nodes, but some (likr mesos-agent) run on the agent nodes.
```
Comment (drewvanstone) [inline pages/1.13/overview/concepts/index.md]: mind fixing this typo `s/likr/like/g`

File: pages/1.13/overview/concepts/index.md [hunk 4]
```diff
@@ -242,7 +242,7 @@ A DC/OS component is a DC/OS system service that is distributed with DC/OS.
...
 
-**Examples:** Mesos, Marathon, Mesos-DNS, Bouncer, Admin Router, DC/OS Package Manager (Cosmos), History Service, etc.
+**Examples:** Mesos, Marathon, Mesos-DNS, Bouncer, Admin Router, DC/OS Package Manager (Cosmos), History Service (deprecated), etc.
```
Comment (drewvanstone) [inline pages/1.13/overview/concepts/index.md]: Again, probably don't need to add deprecated here. Maybe instead we should just remove `History Service` as one of the examples.

File: pages/1.13/security/ent/perms-reference/index.md [hunk 5]
```diff
@@ -71,7 +71,8 @@ through Admin Router.
...
 | `dcos:adminrouter:ops:exhibitor`<br> Controls access to the Exhibitor UI and API. This permission allows users to [remove the ZooKeeper state](/1.13/deploying-services/uninstall/#framework-cleaner) after uninstalling a service.                                                                     | x      |     |     |     |     |
-| `dcos:adminrouter:ops:historyservice`<br>Controls access to the [History Service](/1.13/overview/architecture/components/#dcos-history). This includes access to potentially sensitive data from Mesos such as the names of all frameworks, its used resources, and the number of tasks in each status. | x      |     |     |     |     |
+| `dcos:adminrouter:ops:historyservice`<br>Controls access to the [History Service](/1.13/overview/architecture/components/#dcos-history). This includes access to potentially sensitive data from Mesos such as the names of all frameworks, its used resources, and the number of tasks in each status. 
+This is deprecated and will be removed in a future version. | x      |     |     |     |     |
```
Comment (drewvanstone) [inline pages/1.13/security/ent/perms-reference/index.md]: This renders in a new row. Don't think there can be a line break

**Comment by janisz** [comment]
@ck4adventure Can you take a look?

**Comment by janisz** [comment]
@pavisandhu  Can you take a look?

**Pull request merged**